.circleci/scripts/maps.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# The map's key and value must be delimited by `:`,
+# Don't leave whitespace between key and value like `10.1: cudnn-10.1-windows10-x64-v7.6.4.38`
+#
+# example
+# declare -a installers=(
+#    "10.1:cudnn-10.1-windows10-x64-v7.6.4.38",
+#    "11.1:cudnn-11.1-windows-x64-v8.0.5.39",
+#    "11.2:cudnn-11.2-windows-x64-v8.1.0.77",
+#    "11.3:cudnn-11.3-windows-x64-v8.2.0.53",
+# )
+#
+# #map_get_value 10.1 "${installers[@]}"
+# my_value=$map_return_value
+
+
+map_get_value() {
```

Comment:
Doesn't this just seem like a roundabout way of doing a `case` statement in bash?

Seems like we could simplify a lot of this by just using `case` statements.

See: https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_03.html

Reply:
So far, switch hasn't much difference from my function.
My goal is to reduce the maintenance effort step by step.
In the long run, all those hard codes, which have to be changed manually, should all be put in the configuration.
It's easier to update them or make the change automatically.